packages/js-multiline-to-singleline/src/index.ts

```diff
@@ -81,7 +81,7 @@ export function makeMultilineJSIntoSingleLine(src: string): string {
   // ASI and *only* ASI and leaves the source intact otherwise.
   let postASI: string;
   try {
-    // eslint-disable-next-line no-sync
+     
```

Comment:
I'm.. going to leave this one for prettier rather than run all of CI again right now 😆